site/gatsby-site/src/components/AiidHead.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import Helmet from 'react-helmet';
 import config from '../../config';
 
-const AiidHelmet = ({
+const AiidHead = ({
   children,
   metaTitle = null,
-  metaDescription = null,
+  metaDescription = metaTitle,
   path,
   metaImage = null,
   metaType = 'website',
+  title = null,
```

Comment:
We should use either `title` or `metaTitle` but not both since they refer to the same thing.